repos/system_upgrade/common/actors/addupgradebootentry/libraries/addupgradebootentry.py

```diff
@@ -11,8 +11,8 @@ def add_boot_entry(configs=None):
     debug = 'debug' if os.getenv('LEAPP_DEBUG', '0') == '1' else ''
 
     kernel_dst_path, initram_dst_path = get_boot_file_paths()
+    _remove_old_upgrade_boot_entry(kernel_dst_path, configs=configs)
```

Comment:
why the move out of the try block?

Reply:
We are handling possible exception inside the `_remove_old_upgrade_boot_entry` function while the previous try block was handling and logging potential issues with configuring the bootloader, namely showing `'Cannot configure bootloader.'`.